README.md

```diff
@@ -30,7 +30,7 @@ import Model from 'ember-data/model';
 import { modelAction } from 'ember-custom-actions';
 
 export default Model.extend({
-  publish: modelAction('publish', { pushToStore: false }),
+  publish: modelAction('publish', { pushToStore: false, extractInvalidError: false }),
```

Comment:
I like the split in three methods. :)
Thanks for teaching me how to use rethrow.